src/main/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/auditing/MapLocationMetadata.kt

```diff
@@ -0,0 +1,56 @@
+package uk.gov.justice.digital.hmpps.pecs.jpc.auditing
+
+import com.beust.klaxon.Json
+import com.beust.klaxon.Klaxon
+import uk.gov.justice.digital.hmpps.pecs.jpc.location.Location
+import uk.gov.justice.digital.hmpps.pecs.jpc.location.LocationType
+
+/**
+ * Metadata to capture mapping of new locations and remapping of existing locations.
+ */
+data class MapLocationMetadata(
```

Comment:
So I have created a dedicated class to represent the location changes/audits as this will make it much easier to work with and use in the code for presentation purposes.